app/javascript/components/widgets/fires/fires-alerts-cumulative/selectors.js

```diff
@@ -303,7 +303,7 @@ export const parseConfig = createSelector(
         })
       },
       legend,
-      tooltip,
+      tooltip: [...tooltip, { sortBy: 'value' }],
```

Comment:
This doesn't make a lot of sense to me. Tool tip is an array of configs for rendering, but the final item is how to order them? We need to be more explicit if we are going to make changes to configs like these used over so many components.

Possibly an additional key is required.

Reply:
I guess we could add something like `tooltipConfig`, I agree we need to find something standard for the configs.